Services/Queue/Processors/IndexPagesProcessor.php

```diff
@@ -38,6 +38,9 @@ public function __construct(
 
     public function execute(array $data, $storeId)
     {
+        if ($this->storeAutocompleteConfig->hasToSyncPages()) {
```

Comment:
```suggestion
        if (!$this->storeAutocompleteConfig->hasToSyncPages()) {
```

We need to return from this function is it false.